style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background: #191765;
+  color: white;
+  font-family: "Roboto", sans-serif;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+/*
+  Header
+*/
+.nav {
+  flex: 1;
+}
+
+.wrapper .nav ul {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.wrapper .nav ul .name {
+  flex: 6;
+}
+
+.wrapper .nav ul > li {
+  list-style: none;
+  flex: 1;
+}
+
+.wrapper .nav ul .mail {
+  flex: 1;
+}
+
+/*
+  Headline Section
+*/
```

Comment:
- [ ] Kindly remove comments as code, you did a nice job naming your class in a descriptive way, therefore is no need for the comments